src/main/java/org/mskcc/limsrest/service/PromoteBanked.java

```diff
@@ -417,6 +417,16 @@ public void createRecords(DataRecord bankedSampleRecord, DataRecord req, String
             Object requestedCoverage = bankedFields.getOrDefault("RequestedCoverage", null);
             String bankedSampleRequestedReads = bankedSample.getRequestedReads();
 
+            //Populating number of amplicons in MissionBioTapestri lib prep Protocol1 table
+            //Runs if recipe is a MissionBio kind.
+            if(recipe.toLowerCase().contains("missionbio")) {
```

Comment:
need to add something like:
`&& bankedSample.getSampleType() == "cells" || bankedSample.getSampleType() == "nuclei"`

Reply:
Added.